scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/package.scala

```diff
@@ -1,3 +1,5 @@
+package com.twitter.scalding
```

Comment:
I think `reducer_estimation` package object should be in `com.twitter.scalding` package.
http://www.scala-lang.org/docu/files/packageobjects/packageobjects.html
http://docs.scala-lang.org/style/naming-conventions.html#packages